nodemon.json

```diff
@@ -1,6 +1,6 @@
 {
   "watch": ["app"],
   "ext": "ts, json",
-  "ignore": ["src/**/*.spec.ts"],
+  "ignore": ["src/**/*.spec.ts", "app/locales/en.json"],
```

Comment:
Maybe change to `app/locals/*.json`

Reply:
It was because that file is auto-generated and would cause the server to restart, we can remove it when `updateFiles: false`

Reply:
Nice, feel free to add it 👍

Reply:
Removed it now.

Reply:
Added!